dev-db/etcd-wrapped/etcd-wrapped-0.4.9.ebuild

```diff
@@ -0,0 +1 @@
+files/etcd-wrapped.ebuild
```

Comment:
So can I remove all workarounds/efforts for etcd0 from the PR? This also means no more `etcd.service` on CoreOS.